lessons/misc/ml-intro/kmeans.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: Unsupervised Learning
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+rm(list = ls())
+```
+
+## GitHub Documents
+
+This is an R Markdown format used for publishing markdown documents to GitHub. When you click the **Knit** button all R code chunks are run and a markdown file (.md) suitable for publishing to GitHub is generated.
+
+## Including Code
+
+You can include R code in the document as follows:
+
+```{r iris}
+library(datasets)
+head(iris)
+```
+
+## Including Plots
+
+You can also embed plots, for example:
+
+```{r pressure, echo=FALSE}
+first<-plot(iris)
```

Comment:
space around `<-`. If you use RStudio, you can use `Ctrl-Shift-A` to reformat and clean up the code.